### PR TITLE
Add new endpoints for inserting suggestion ratings

### PR DESCRIPTION
**Pull request by fabubaker**
The following are the new endpoints:
'/insert-suggestion-rating'
'/batch-insert-suggestion-rating'

A new column 'suggestion_ratings' is also added to the review table.

### CODE REVIEW

File: client/run.rb [hunk 1]
```diff
@@ -456,3 +456,15 @@ def to_hash
 
 res = rayyan_store.batch_update_dedup_user_action(review_id, dedup_updates)
 puts "Batch update returned with: #{res}"
+
+res = rayyan_store.insert_suggestion_rating(review_id, article_id_to_del, 1.0, 3)
+puts "Insert suggestion rating returned with: #{res}"
+
+inserts_list = article_ids.sample(5).map{|id| {
+  article_id: id,
+  rating_value: [0.1, 0.25, 0.5, 0.75, 1.0].sample(1)[0],
```
Comment (hammady) [inline client/run.rb]: use .sample without params

File: client/gem/lib/rayyan-store.rb [hunk 2]
```diff
@@ -136,6 +136,26 @@ def batch_update_dedup_user_action(review_id, updates_list)
       }
     end
 
+    def insert_suggestion_rating(review_id, article_id, rating_value, star_rating)
```
Comment (hammady) [inline client/gem/lib/rayyan-store.rb]: Rename to `score` and `stars`.

File: server/src/adapters/postgres.js [hunk 3]
```diff
@@ -1023,5 +1024,42 @@ module.exports = {
       `
 
     return pool.query(dedupQuery, [JSON.stringify(updatesList)])
+  },
+
+  insertSuggestionRating: function (reviewId, articleId, ratingValue, starRating) {
```
Comment (hammady) [inline server/src/adapters/postgres.js]: Rename params here as well.

File: server/src/adapters/postgres.js [hunk 4]
```diff
@@ -1023,5 +1024,42 @@ module.exports = {
...
+    // @brief Performs a batch insert of suggestion ratings.
+    // @param {String} reviewId - The id of the review to insert ratings.
+    // @param {Array} insertsList - Array of Dicts of the form
+    //                              {article_id: <article_id>,
+    //                               rating_value: <rating_value>,
+    //                               star_rating: <star_rating>}
+
+    const batchQuery =
+      `
+      WITH insertion as (SELECT * FROM jsonb_array_elements($1))
```
Comment (hammady) [inline server/src/adapters/postgres.js]: Nice!

File: client/run.rb (hunk 1 above)
Comment (fabubaker) [reply client/run.rb]: Fixed in e1cf988.

File: client/gem/lib/rayyan-store.rb (hunk 2 above)
Comment (fabubaker) [reply client/gem/lib/rayyan-store.rb]: Fixed in 44ec9e7

File: server/src/adapters/postgres.js (hunk 4 above)
Comment (fabubaker) [reply server/src/adapters/postgres.js]: Fixed in 44ec9e7

File: server/src/adapters/postgres.js (hunk 3 above)
Comment (fabubaker) [reply server/src/adapters/postgres.js]: Fixed in 44ec9e7

File: server/src/adapters/postgres.js (hunk 4 above)
Comment (fabubaker) [reply server/src/adapters/postgres.js]: :smile:

**Pull request merged**